.claude/skills/observability-instrumentation/SKILL.md

```
@@ -1,593 +1,586 @@
-        ]);
-
-        $this->incrementCounter('customer.create.errors');
-
-        throw $e;
-    }
-}
-```
+## Type-Safe Metric Class Hierarchy
 
-**Key metrics to track**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)